go/logic/migrator.go

```diff
@@ -734,6 +746,17 @@ func (this *Migrator) initiateServer() (err error) {
 	return nil
 }
 
+// attemptInstantDDL tries to apply the DDL statement directly to the table
+// using a ALGORITHM=INSTANT assertion. If this fails, it will return an error,
+// in which case the original algorithm should be used.
+func (this *Migrator) attemptInstantDDL() (err error) {
+	this.applier = NewApplier(this.migrationContext)
```

Comment:
@morgo assuming my earlier suggestion is made I'd like to avoid opening the applier twice, just in case

I'm hoping the applier setup by `[this.initiateApplier()](https://github.com/github/gh-ost/blob/master/go/logic/migrator.go#L357-L359)` in `.Migrate()` can be used here